orchestrator/orchestrator.go

```diff
@@ -163,7 +167,7 @@ func (orch *Orchestrator) startSiteWatcher(site site, close chan struct{}) {
 		orch.logger.Debugf("initial delay has elapsed for %+v, starting to run events", site)
 	}
 
-	ticker := time.NewTicker(orch.config.GetEventsInterval)
+	ticker := jitterbug.New(orch.config.GetEventsInterval, &jitterbug.Norm{Stdev: orch.config.GetEventsInterval / 20})
```

Comment:
we use 5% of the mean as the stdev, which, for the default value of 15s, gives a stdev of 750ms and the following PDF:

![image](https://user-images.githubusercontent.com/5695343/143097004-5da52a61-dd13-4717-9543-1e2f4c5799d1.png)